cedar-policy-cli/src/lib.rs

```diff
@@ -935,25 +928,6 @@ fn read_schema_file(filename: impl AsRef<Path> + std::marker::Copy) -> Result<Sc
         })
 }
 
-fn load_actions_from_schema(entities: Entities, schema: &Option<Schema>) -> Result<Entities> {
```

Comment:
Removing this function has it will be incorporated into the main cedar library, and we want a very specific error semantics for it

Reply:
This overlaps with my #360

Reply:
but I think it's the same change, so we should be able to accept both in either order